backend/db/load.py

```diff
@@ -25,44 +60,43 @@ def download_artefacts(force_download_if_exists=False):
     download_favorite_datasets(force_if_exists=force_download_if_exists, single_group='vocab')
 
 
-def seed(con: Connection, schema: str = SCHEMA, clobber=False, skip_if_updated_within_hours: int = None):
+def initialize_test_schema(con: Connection, schema: str = SCHEMA, local=False):
```

Comment:
@Sigfried New func `initialize_test_schema()`. Runs during `initialize()`, but I ran it ad hoc to set up `test_n3c` for prod and my local.